eureka2-ext/eureka2-aws/src/main/java/com/netflix/eureka2/ext/aws/S3OverridesConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.eureka2.ext.aws;
+
+import com.netflix.archaius.annotations.DefaultValue;
+
+/**
+ * @author David Liu
+ */
+public interface S3OverridesConfiguration extends AwsConfiguration {
```

Comment:
Does it make sense to have single AwsConfiguration interface for all AWS configuration data?